src/app/core/navbar.service.ts

```diff
@@ -0,0 +1,29 @@
+/**
+   * Service that allows components to show or hide the navbar
+   */
+
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class NavbarService {
+  visible: boolean;
+
+  constructor() {
+    this.visible = true;
+  } // navbar is visible by default
+
+  // hide navbar
+  hide() {
```

Comment:
Using a setTimeout here is a hacky fix, without it the code ran too soon and angular was ignoring it. This will be replaced with a more permanent solution